HIPs/hip-1.md

```diff
@@ -0,0 +1,271 @@
+
+## HIP Rationale
+
+We intend HIPs to be the primary mechanisms for proposing new features, for collecting community technical input on an issue, and for documenting the design decisions that have gone into Handshake Because the HIPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+For Handshake client implementers, HIPs are a convenient way to track the progress of their implementation. Ideally each implementation maintainer would list the HIPs that they have implemented. This will give end users a convenient way to know the current status of a given implementation or library.
+
+## HIP Types
+
+There are three types of HIP:
+
+- A **Standard Track HIP** describes any change that affects most or all Handshake implementations, such as a change to the network protocol, a change in block or transaction validity rules, proposed naming application standards/conventions, or any change or addition that affects the interoperability of applications using Handshake. Furthermore Standard HIPs can be broken down into the following categories. Standards Track HIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification].
+  - **Core** - improvements requiring a consensus fork, as well as changes that are not necessarily consensus critical but may be relevant to Handshake Client Developer discussions
+  - **Networking** - includes improvements around P2P networking layer and tbe HNSD light client protocol, as well as proposed additional improvements to any relevant networked protocol specifications.
+  - **Interface** - includes improvements around client [API/RPC] specifications and standards, and also certain language-level standards like method names. The label “interface” aligns with the [interfaces repo] and discussion should primarily occur in that repository before an HIP is submitted to the HIPs repository.
+  - **HRC** - application-level standards and conventions, including contract standards such as naming and domain standards, name registries, URI schemes, library/package formats, and wallet formats.
+- A **Meta HIP** describes a process surrounding Handshake or proposes a change to (or an event in) a process. Process HIPs are like Standards Track HIPs but apply to areas other than the Handshake protocol itself. They may propose an implementation, but not to Handshake's codebase; they often require community consensus; unlike Informational HIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Handshake  development. Any meta-HIP is also considered a Process HIP.
```

Comment:
Let's pick either "meta" or "process" and just go with it instead of having a synonym - this is a specification after all!